lib/ansible/modules/cloud/amazon/elb_instance.py

```diff
@@ -86,34 +86,36 @@
     with_items: "{{ ec2_elbs }}"
 """
 
-import time
```

Comment:
Please move porting this module to its own PR. Every PR porting a module to boto3 must also have accompanying integration tests to be able to be merged.

Modules marked `stableinterface` must keep input and return data the same for backward compatibility, where you have a little more leeway with modules marked `preview` (but should document anything that doesn't keep the same behavior that it had before).